lib/web_ui/lib/src/engine/surface/recording_canvas.dart

```diff
@@ -309,7 +309,8 @@ class RecordingCanvas {
     }
     _paintBounds.grow(pathBounds);
     // Clone path so it can be reused for subsequent draw calls.
-    final ui.Path clone = ui.Path.from(path);
+    final ui.Path clone = (experimentalUseSkia) ?
```

Comment:
The `experimentalUseSkia` check here should be unnecessary. The CanvasKit renderer should never touch `recording_canvas.dart`

Reply:
Done, calling SurfacePath directly now. Thx.